docs/devops-documentation.md

```diff
@@ -38,6 +38,8 @@ The main configuration file is `.env`, located in root folder.
 | CACHE_TTL_LTI_INSTANCES | Cache TTL (in seconds) for LTI instance list [default: `3600`] |
 | CORS_ALLOW_ORIGIN | Allowed CORS origin |
 | ASSIGNMENT_STATE_INTERVAL_THRESHOLD | Threshold for assignment garbage collection [default: `P1D`] |
+| WEBHOOK_USERNAME | Webhook endpoint username |
+| WEBHOOK_PASSWORD | Webhook endpoint password |
```

Comment:
Proposal: since the basic auth is not really coupled with webhooks (for now it is, but who knows how it would be used later) - change naming to smth like `BASIC_AUTH_USERNAME` / `BASIC_AUTH_PASSWORD`

Reply:
Agreed and renamed